.nvmrc

```diff
@@ -1 +1 @@
-8.9.4
+10
```

Comment:
How come, govuk-frontend have set there's to 10.15.1

Reply:
eslint 5 doesn't work on 8.9.4.

'10' installs latest version of 10 ensuring security patches are run on CI

10 is currently LTS